src/extension/options-page/DashboardComponents/ProviderLine.tsx

```diff
@@ -1,56 +1,73 @@
-import { FormControl, Typography, Theme } from '@material-ui/core'
 import React from 'react'
-import { makeStyles, createStyles } from '@material-ui/styles'
+import { Typography, Divider } from '@material-ui/core'
+import { makeStyles, createStyles } from '@material-ui/core/styles'
 import classNames from 'classnames'
 import { useI18N } from '../../../utils/i18n-next-ui'
 
-const useStyles = makeStyles((theme: Theme) =>
+import LinkOffIcon from '@material-ui/icons/LinkOff'
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward'
+
+const useStyles = makeStyles(theme =>
     createStyles({
         text: {
-            width: '200px',
-            display: 'block',
             color: theme.palette.primary.main,
-        },
-        control: {
             display: 'flex',
-            textAlign: 'left',
-            width: '100%',
+            alignItems: 'center',
+            margin: theme.spacing(1.2, 2),
+            '& > :first-child': {
+                flex: '1 1 auto',
+                overflow: 'hidden',
+                whiteSpace: 'nowrap',
+                textOverflow: 'ellipsis',
+            },
+            '& > :last-child': {
+                flex: '0 0 auto',
+            },
         },
         connected: {
             color: theme.palette.text.primary,
         },
-        pointer: {
+        cursor: {
             cursor: 'pointer',
         },
+        control: {
+            marginTop: theme.spacing(2),
+        },
     }),
 )
 
 interface ProviderLineProps {
     network: string
     connected?: boolean
     userId?: string
-    onConnect?: () => void
+    onAction?: () => void
```

Comment:
Isn't this name too ambigious?

Reply:
maybe an alias of `onClick`... or `onTrigger`